src/hpr_lorawan.erl

```diff
@@ -0,0 +1,36 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 18. Sep 2022 12:22 PM
+%%%-------------------------------------------------------------------
+-module(hpr_lorawan).
+-author("jonathanruttenberg").
+
+%%%-------------------------------------------------------------------
+%% @doc
+%% Copyright (c) 2016-2019 Petr &lt;Gotthard petr.gotthard@@centrum.cz&gt;
+%% All rights reserved.
+%% Distributed under the terms of the MIT License. See the LICENSE file.
+%% @end
+%%%-------------------------------------------------------------------
+
```

Comment:
```suggestion
```

We can remove this. The original file was based off Petr's work, but none of that exists any longer.